test/evergreen.yml

```diff
@@ -2291,7 +2291,8 @@ buildvariants:
     - name: ftruncate-test
     - name: long-test
     - name: configure-combinations
-    - name: coverage-report
+    # Temporarily disabled
```

Comment:
Double checked with luke and we were suppose to disable coverage report, when syncing with develop branch.